src/button/Link.module.css

```diff
@@ -0,0 +1,10 @@
+/*
+Copyright 2024 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only
+Please see LICENSE in the repository root for full details.
+*/
+
+.external {
+  color: inherit;
+}
```

Comment:
Why is this needed. What is the color property set to otherwise.

Would a comment make sense here?